### PR TITLE
Fix building tvOS apps

### PR DESCRIPTION
**Pull request by tmm1**
This lets `gym` build tvos apps

### CODE REVIEW

File: lib/gym/detect_values.rb [hunk 1]
```diff
@@ -62,7 +62,7 @@ def self.detect_scheme
     # Is it an iOS device or a Mac?
     def self.detect_platform
       return if Gym.config[:destination]
-      platform = Gym.project.mac? ? "OS X" : "iOS" # either `iOS` or `OS X`
+      platform = Gym.project.mac? ? "OS X" : Gym.project.tvos? ? "tvOS" : "iOS"
```
Comment (KrauseFx) [inline lib/gym/detect_values.rb]: Too many `?` in one line, please change that to multi-line

**Comment by KrauseFx** [comment]
Hey @tmm1, thanks for the PR. Did you test `gym` with these changes to build tvOS apps? Have you tried submitting an app to the store to see if it passes validation?

**Comment by tmm1** [comment]
Yes, submitted through TestFlight without a problem.

The only issue I've noticed is that it looks like all files are rebuilt every time I run gym, even though `clean false` is set. Not sure if that's expected behavior or some sort of regression with tvos.

File: lib/gym/detect_values.rb (hunk 1 above)
Comment (tmm1) [inline lib/gym/detect_values.rb]: Multiline ternary, or if/else?

Comment (KrauseFx) [inline lib/gym/detect_values.rb]: Yeah, if else, or case :+1:

**Comment by KrauseFx** [comment]
@tmm1 Yeah, that's a separate issue with `xcodebuild` :+1:

**Comment by tmm1** [comment]
> @tmm1 Yeah, that's a separate issue with xcodebuild

Bummer. Got any links handy?

**Comment by KrauseFx** [comment]
Could you squash the commits, before I can merge it :+1:

**Comment by tmm1** [comment]
Done.

**Comment by tmm1** [comment]
Ping.

**Comment by KrauseFx** [comment]
Sorry for the delay, and thanks for updating the PR @tmm1 :+1:

**Comment by mgazdich** [comment]
I saw this was merged into master however I am still seeing the "gym doesn't suppoort tvOS projects yet, we're working on adding this feature!". Is there still some development going on for this feature, or is something else wrong, and i should be able to make build for tvOS?

**Comment by KrauseFx** [comment]
@mgazdich Yeah, I just noticed I haven't pushed a new release in a while, will do that right now :+1:

**Comment by KrauseFx** [comment]
It's live :rocket: https://github.com/fastlane/gym/releases/tag/1.5.0, thanks again for letting me know @mgazdich :+1:

**Comment by mgazdich** [comment]
Absolutely! I just ran an update. One little thing I am notice however, is when i have the default platform specified in my fastfile as  tvos is gives me the same "doesnt support" error. 

Any incite? And if there is a better avenue to communicating with you, by all means.

**Comment by KrauseFx** [comment]
@mgazdich Please submit a separate GitHub issue and share your `Fastfile` and the complete output when running it :+1:

**Pull request merged**